packages/medulas-react-components/package.json

```diff
@@ -15,15 +15,7 @@
     "not op_mini all"
   ],
   "dependencies": {
-    "@material-ui/core": "4.0.0-alpha.6",
```

Comment:
Those dependencies belong here. Otherwise a user of medulas-react-components cannot use medulas-react-components due to missing dependencies.

Reply:
The main reason because I extracted them was not having duplicated versions of the same library, do you know if it is possible to create constants?

Reply:
Since .json does not allow coding, the only way I see is to use a code generator that updates all `package.json`s. However, it is much simpler to search the whole project for "@material-ui/core" (including the quotes) via CMD+Shift+F and update 2 or 3 places.